tool/firestore-emulator/Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG GOOGLE_CLOUD_SDK_VERSION=392.0.0-alpine
 
-FROM google/cloud-sdk:$GOOGLE_CLOUD_SDK_VERSION
+FROM gcr.io/google.com/cloudsdktool/google-cloud-cli:$GOOGLE_CLOUD_SDK_VERSION
```

Comment:
Container image was changed. Because `google/cloud-sdk` doesn't support arm64.
https://cloud.google.com/sdk/docs/downloads-docker

```console
$ uname -m
arm64

$ docker inspect google/cloud-sdk:392.0.0-alpine | jq -r .[].Architecture
amd64

$ docker inspect gcr.io/google.com/cloudsdktool/google-cloud-cli:392.0.0-alpine | jq .[].Architecture
arm64
```